compute/compute-node.Dockerfile

```diff
@@ -1482,7 +1493,7 @@ RUN set -ex; \
     else \
         echo "Unsupported architecture: ${TARGETARCH}"; exit 1; \
     fi; \
-    curl -L "https://awscli.amazonaws.com/awscli-exe-linux-${TARGETARCH_ALT}-2.17.5.zip" -o /tmp/awscliv2.zip; \
+    curl --retry 5 -L "https://awscli.amazonaws.com/awscli-exe-linux-${TARGETARCH_ALT}-2.17.5.zip" -o /tmp/awscliv2.zip; \
     echo "${CHECKSUM}  /tmp/awscliv2.zip" | sha256sum -c -; \
```

Comment:
I believe we should also add retries to the newly added exporters layer https://github.com/neondatabase/neon/blob/34e560fe373821114049dd97144d80d31171b13b/compute/compute-node.Dockerfile#L1286, referencing my PR as this one is based on an older main

Reply:
discussed in slack and agreed I will add here after merging that PR